src/pageNavigation.js

```diff
@@ -148,7 +147,12 @@ async function addListener(listener, {
 
     // Compile the match patterns for the listener
     const matchPatternSet = matching.createMatchPatternSet(matchPatterns);
-    const contentScript = await contentScripts.registerContentScript(matchPatterns, pageNavigationContentScript);
+    const contentScript = await browser.scripting.registerContentScripts([{
```

Comment:
nit: on line 38 of this file, the @property type should now be changed from `browser.contentScripts.RegisteredContentScript` to `browser.scripting.RegisteredContentScript`

Reply:
see: https://github.com/mozilla-rally/web-science/blob/75f8515b45c75d3d7cfda651cd7b6e7424e1600e/src/pageNavigation.js#L38